src/Api/Model/Shared/UserModel.php

```diff
@@ -161,7 +160,7 @@ public function __construct($id = "")
     /** @var string */
     public $mobile_phone;
 
-    /** @var string - possible values are "email", "sms" or "both" */
+    /** @var string - possible values are "email" or "both" */
     public $communicate_via;
```

Comment:
In fact this whole property should be gone.

```suggestion
```

Reply:
yep!  [536d630](https://github.com/sillsdev/web-languageforge/pull/1611/commits/536d630a7185e7c219448dd7853b224a325392a5)

Reply:
thanks!